src/app/modules/group/components/group-manager-list/group-manager-list.component.ts

```diff
@@ -42,8 +42,8 @@ export class GroupManagerListComponent implements OnChanges {
         this.managers = managers.map(manager => ({ ...manager, can_manage_as_text: this.getManagerLevel(manager) }));
         this.state = 'ready';
       },
-        _err => {
-          this.state = 'error';
-        });
+      _err => {
```

Comment:
Here, it is not really correct, but mainly as if `_err =>..` is on a new line, `(managers: Manager[]) => ` should be on a new line as well

Reply:
fixed